### ISSUE TITLE
plugins:add fails silently if yarn fails

### ISSUE THREAD

**Issue by dprothero** [oclif/plugin-plugins#73]
Using this CLI: https://gitub.com/twilio/twilio-cli

running: `twilio plugins:add @twilio-labs/plugin-serverless`

on a machine with Node less than 8.10 (my use case is 8.9.1) will just fail silently after step 2/4

When I copy and manually run the yarn command that is shown via DEBUG=* I see the issue:

```
yarn add v1.15.2
[1/4] Resolving packages...
[2/4] Fetching packages...
error @twilio-labs/plugin-serverless@1.0.0-alpha.2: The engine "node" is incompatible with this module. Expected version ">=8.10.0". Got "8.9.1"
error Found incompatible module
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

Expected behavior: I should see the yarn error output instead of nothing when installing a plugin and it fails. Or, at least a "plugin installation failed" and the full details when using DEBUG.

**Comment by dprothero** [comment]
Also fails if the plugin cannot be found:
```
➜  ~ twilio plugins:install lasdjkfiwejfojdslfkjsdlfjie
Installing plugin lasdjkfiwejfojdslfkjsdlfjie... ⣯ [1/4] Resolving packages...
➜  ~
```

**Comment by dprothero** [comment]
@RasPhilCo any thoughts on this one?

**Comment by RasPhilCo** [comment]
Hey @dprothero, I'm back from PTO. Looks like the error message is getting swallowed if cli.action is running. Will need to investigate further.

**Comment by RasPhilCo** [comment]
Should be fixed in #195
